sdk/daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Blinding.scala

```diff
@@ -99,7 +100,7 @@ object Blinding {
       contractVisibility: Relation[ContractId, Party],
   ): Iterable[(Party, PackageId)] = {
     for {
-      (contractId, packageId) <- contractPackages
+      (contractId, packageId) <- contractPackages.view
```

Comment:
This fixes a bug: Without the `.view`, the for comprehension ends up deduplicating party -> packageId associations prematurely